r_markdown/emissions.Rmd

```diff
@@ -442,13 +548,6 @@ ___
   ions <- dplyr::mutate(ions, 
                         mass_emitted = (conc - conc_bg) * flow_hood * dur,
                         mass_carbon = mass_emitted * 0)
-
-  ions_g <- dplyr::group_by(ions, id) %>%
```

Comment:
I've add a new variable "inst" to emissions.Rmd  in the branch fix_temp_energy so you can group by instrument

This is what you would have done? Please, please please?